### DISCUSSION TITLE
How to create circular hollow sections

### DISCUSSION THREAD

**Discussion by Rockambole** [idea-statica/ideastatica-public#439] [Help Q&A] [answered]
Dear Nathan,

With good help from yourself, Martin and David, I have managed to overcome the license issues, so now I can focus on the task at hand: transfer FE data from our analysis tool SESAM, to IDEAStatica…

Geometry and loads seem to be in place, however, I need to be able to define cross sections for IOM export. The following Vb.Net snippet, derived from one of the IOM examples looks like:

 
![image](https://github.com/idea-statica/ideastatica-public/assets/16279406/82db5108-e427-4f75-ad8f-8a2360eb551e)

Would you have a manual, or more comprehensive examples, to define box, tubulars etc. I can’t seem to find any…

Thanks,

Regards, Odd

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @Rockambole,

You can find some information here. https://developer.ideastatica.com/docs/iom/iom_cross_sections.html#rolled-sections

You should use the CrossSectionFactory methods available to create these cross-sections. Here is a link to the method you should use to fill a CHS section. https://developer.ideastatica.com/api-iom/IdeaRS.OpenModel.CrossSection.CrossSectionFactory.html#IdeaRS_OpenModel_CrossSection_CrossSectionFactory_FillRolledCHS_IdeaRS_OpenModel_CrossSection_CrossSectionParameter_System_Double_System_Double_

Cheers
Nathan
